Source/WebCore/rendering/svg/RenderSVGInline.cpp

```diff
@@ -79,8 +79,6 @@ LayoutRect RenderSVGInline::clippedOverflowRect(const RenderLayerModelObject* re
 #if ENABLE(LAYER_BASED_SVG_ENGINE)
     if (document().settings().layerBasedSVGEngineEnabled())
         return RenderInline::clippedOverflowRect(repaintContainer, context);
-#else
```

Comment:
While this seems unnecessary since context is used on the next real line, the error message I see from https://munchkin.rd.scei.sony.co.jp/jenkins/job/Neko.trunk.PPR.Release/8604/console seems to imply the error is on line 94 which is the UNUSED_PARAM in the next method not this one.